API/Entities/Operation.cs

```diff
@@ -1,21 +1,33 @@
 using System;
+using System.ComponentModel.DataAnnotations;
 
 namespace API.Entities
 {
     public class Operation
     {
+        public int Id { get; set; }
+
+        [Required]
+        [MaxLength(100)]
+        public string Description { get; set; }
+        
+        [Required]
         public double Amount { get; set; }
```

Comment:
Do pieniazkow uzywa sie decimala a nie doubla. Go tez mozna ograniczyc jak stringa (precyzje)

Reply:
Poza tym to double wiec nie ma sensu dawac mu required bo on i tak nie przyjmuje nulla wiec ef go uzna  za required automatycznie